examples/charlcd_mono_simpletest.py

```diff
@@ -53,7 +54,7 @@
 scroll_msg = '<-- Scroll'
 lcd.message = scroll_msg
 # Scroll message to the left
-for i in range(len(scroll_msg)):
+for i in range(len(scroll_msg)+5):
     time.sleep(0.5)
```

Comment:
Looks like there are still some changes to the simpletest, so just remove the +5